interview-questions.md

```diff
@@ -8,62 +8,55 @@ Then, research the question to expand on your answer. Even if you feel you have
 
```

Comment:
You certainly have a great understanding of the elements of web development. Now let's add more of your experience with each element so the interviewer and get to know you.